File: solutions/lua/darts/1/darts.lua
local Darts = {}

function Darts.score(x, y)
    local spot = x * x + y * y
    local points = 0
    if     ( spot <=   1 ) then points = 10
    elseif ( spot <=  25 ) then points =  5
    elseif ( spot <= 100 ) then points =  1
    end 
    return points
end

return Darts

File: solutions/erlang/darts/1/src/darts.erl
-module(darts).

-export([score/2]).


score(_X, _Y) -> 
    Spot = _X * _X + _Y * _Y,
    if 
        Spot =<   1 -> 10;
        Spot =<  25 ->  5;
        Spot =< 100 ->  1;
        true -> 0
    end
.
